src/felleskomponenter/etiketer/Etiketter.module.less

```diff
@@ -27,3 +27,9 @@
     background-color: @redErrorLighten80;
     border: 1px @redErrorLighten20 solid;
 }
+
+.samtalereferatIkkeDelt {
+    background-color: #ffd399;
+    border: 1px solid #d87e0a;
```

Comment:
Finnes det noen konstanter vi kan bruke for disse fargene her? 😊

Reply:
Tror ikke det. Dette er slik tilsvarende label er definert i `aktivitetsplan`.